docs/hub/datasets-webdataset.md

```diff
@@ -18,6 +18,21 @@ Labels and metadata can be in a `.json` file, in a `.txt` (for a caption, a desc
 A large scale WebDataset is made of many files called shards, where each shard is a TAR archive.
 Each shard is often ~1GB but the full dataset can be multiple terabytes!
 
+## Multimodal support
+
+WebDataset is designed for multimodal datasets, i.e. for image, audio and/or video datasets.
+
+Indeed since media files tend to be quite big, the sequential I/O of WebDataset enables large reads and buffering. This results in obtaining the best data loading speed.
```

Comment:
```suggestion
Indeed, since media files tend to be quite big, WebDataset's sequential I/O enables large reads and buffering, resulting in the best data loading speed.
```